embulk-executor-mapreduce/src/main/java/org/embulk/executor/mapreduce/MapReduceExecutor.java

```diff
@@ -384,6 +384,12 @@ public AttemptState getAttemptState()
                 builder.add(new AttemptReport(aid, state));
             } catch (EOFException ex) {  // plus Not Found exception
                 builder.add(new AttemptReport(aid, null));
+            } catch (IOException ex) {
```

Comment:
I have a gut feeling that this IOException is from ObjectMapper#readObject(). Empty content? This catch clause hides too many exceptions from EmbulkMapReduce.readAttemptStateFile() so please identify the cause and catch the exception exactly we're going to ignore.